apps/docs/src/app/platform/documentation/platform-documentation.component.scss

```diff
@@ -15,6 +15,8 @@
     padding-bottom: 2rem;
     background-color: #ffffff;
     padding-top: 52px;
+    overflow-x: hidden;
```

Comment:
is this for the `horizontal scrollbar in 320px resolution` issues? Usually for this issue we get @puru-hk to check on an actual mobile device if it is working fine. 
I am not sure if doing this will adversely affect something somewhere.

Reply:
Tested in actual device (Samsung Galaxy S10 Plus) horizontal scroll bar is not coming in netlify master.

Reply:
in that case, this change can be reverted, right @Lokanathan-k ?